src/components/PageSections/Footer/index.js

```diff
@@ -19,9 +19,9 @@ import { threeoneonePropTypes } from 'components/PageSections/ThreeOneOne/propty
 const Footer = ({ threeoneone, intl }) => (
   <footer>
     <div className="wrapper wrapper--sm container-fluid">
-      <FormFeedback />
+      {/*<FormFeedback />*/}
```

Comment:
I'm just commenting these out for now because we may or may not be including them for OPO launch.